src/utils/result-list.js

```diff
@@ -13,11 +12,7 @@ export function scrollList(resultList, index, delayed = false) {
   const resultEl = resultList[index];
 
   const scroll = () => {
-    if (isFirefox()) {
-      resultEl.scrollIntoView({ block: "center", behavior: "smooth" });
-    } else {
-      chromeScrollList(resultEl, index);
-    }
+    resultEl.scrollIntoView({ block: "center", behavior: "smooth" });
```

Comment:
i think we should keep `chromeScrollList` because otherwise this will crash in when you use the launchpad in chrome.